client/state/stats/emails/utils.js

```diff
@@ -234,30 +235,40 @@ export function parseEmailLinksData( links ) {
 	};
 
 	// filter out links that are not in the stringMap
-	const filteredLinks = links
+	const filteredInternalLinks = internalLinks
 		.filter( ( link ) => stringMap[ link[ 0 ] ] )
 		.sort( ( a, b ) => b[ 1 ] - a[ 1 ] );
 	// Get count of all links where the first element is not a key of stringMap
-	const otherCount = links.reduce( ( count, link ) => {
+	const otherInternalLinksCount = internalLinks.reduce( ( count, link ) => {
 		if ( ! stringMap[ link[ 0 ] ] ) {
 			count += parseInt( link[ 1 ], 10 );
 		}
 		return count;
 	}, 0 );
 
-	const mappedLinks = filteredLinks.map( ( link ) => {
+	const mappedLinks = filteredInternalLinks.map( ( link ) => {
 		return {
 			label: stringMap[ link[ 0 ] ],
 			value: parseInt( link[ 1 ], 10 ),
 		};
 	} );
 
-	if ( otherCount ) {
+	if ( otherInternalLinksCount ) {
 		mappedLinks.push( {
 			label: translate( 'Other', { context: 'Email link type' } ),
-			value: otherCount,
+			value: otherInternalLinksCount,
 		} );
 	}
 
-	return mappedLinks;
+	// add user content links
+	userContentLinks.forEach( ( link ) => {
```

Comment:
If we do something about the previous comment we need to check userContentLinks is array here.